### ISSUE TITLE
Space created between 1st & 2nd level list items

### ISSUE THREAD

**Issue by functionalStoic** [platers/obsidian-linter#75]
Hi @platers,
Thanks for the useful plugin!

I've noticed an issue for a while and wanted to bring it up. I've tried to rule out any other plugins that might affect it

- Test Vault with only the Linter Plugin enabled
- Space created between 1st & 2nd level list items (List item, To-do Item, etc.)
- All rules applied. I've disabled a few that looked like they might apply, but no luck

Here is a screen recording to display the issue

https://user-images.githubusercontent.com/7215306/134734395-afdbea1f-b966-4750-a10f-39298be3e1ca.mp4

Let me know if I can be of any further assistance

**Comment by platers** [comment]
Thanks for the detailed bug report! The bug is in paragraph-blank-line, this is a tricky case when it detects an indented code block when there isn't one.

Will try to fix.

**Comment by functionalStoic** [comment]
@platers Indeed, disabling that rule "solves" the issue for me (trade-offs 😄 ) but obviously not ideal.

Thanks for any effort to fix.
